test/data/stages/systemd/b.json

```diff
@@ -470,6 +470,16 @@
           ],
           "disabled_services": [
             "sshd"
+          ],
+          "presets": [
+            {
+              "name": "ssh.service",
+              "state": "enabled"
```

Comment:
the state for presets it's either `enable` or `disable` (not with the leading `d`) https://www.freedesktop.org/software/systemd/man/systemd.preset.html#Preset%20File%20Format